src/game/common/bitflags.h

```diff
@@ -216,20 +217,218 @@ enum ModelConditionFlagType
 template<int bits> class BitFlags
 {
 public:
+    enum BogusInitType
+    {
+        kInit = 0,
+    };
+
+    BitFlags() {}
+    BitFlags(BogusInitType type, int flag) { m_bits.set(flag); }
+    BitFlags(BogusInitType type, int flag1, int flag2)
+    {
+        m_bits.set(flag1);
+        m_bits.set(flag2);
+    }
+
+    BitFlags(BogusInitType type, int flag1, int flag2, int flag3)
+    {
+        m_bits.set(flag1);
+        m_bits.set(flag2);
+        m_bits.set(flag3);
+    }
+
+    BitFlags(BogusInitType type, int flag1, int flag2, int flag3, int flag4)
+    {
+        m_bits.set(flag1);
+        m_bits.set(flag2);
+        m_bits.set(flag3);
+        m_bits.set(flag4);
+    }
+
+    BitFlags(BogusInitType type, int flag1, int flag2, int flag3, int flag4, int flag5)
```

Comment:
we could do this with a fold expression and handle any number of flags and cut down on all of this repeat code.
```cpp
template<typename... T>
BitFlags(T&&... flags){
    (m_bits.set(flags), ...);
}
```